tests/components/sensor/test_transport_nsw.py

```diff
@@ -0,0 +1,38 @@
+"""The tests for the Transport NSW (AU) sensor platform."""
+import unittest
+
+from homeassistant.setup import setup_component
+
+from tests.common import get_test_home_assistant
+
+VALID_CONFIG = {'sensor': {
+    'platform': 'transport_nsw',
+    'stopid': '209516',
+    'route':  '199',
+    'apikey': 'YOUR_API_KEY'}
+    }
+
+
+class TestRMVtransportSensor(unittest.TestCase):
+    """Test the TransportNSW sensor."""
+
+    def setUp(self):
+        """Set up things to run when tests begin."""
+        self.hass = get_test_home_assistant()
+        self.config = VALID_CONFIG
+        self.reference = {}
+        self.entities = []
+
+    def tearDown(self):
+        """Stop everything that was started."""
+        self.hass.stop()
+
+    def test_transportnsw_config(self):
```

Comment:
Please mock out the api query and result. We don't want to do any I/O in the tests.

Reply:
Fixed